### PR TITLE
chore: unify examples

### PR DESCRIPTION
**Pull request by chromevoid**
simple reduce counter example

### CODE REVIEW

**Comment by vigith** [comment]
should we rename these example repos with the prefix `map-` or `reduce-`?

**Comment by chromevoid** [comment]
> should we rename these example repos with the prefix `map-` or `reduce-`?

the docker image repo?

**Comment by vigith** [comment]
> the docker image repo?
your call 😄

File: pkg/function/examples/counter/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+FROM alpine:3.12.3 as base
+RUN apk update && apk upgrade && \
+    apk add ca-certificates && \
+    apk --no-cache add tzdata
+
+COPY dist/go-counter-example /bin/go-counter-example
+RUN chmod +x /bin/go-counter-example
+
+####################################################################################################
+# evenodd
```
Comment (whynowy) [inline pkg/function/examples/counter/Dockerfile:13]: Comment.

**Comment by whynowy** [comment]
It would be good to put the example images into different groups, for example, `quay.io/numaio/udf/go-counter-example` - feel free to come up with better names.

**Review by vigith — APPROVED** [review]

**Pull request merged**